lib/graphql/subscriptions/anycable_subscriptions.rb

```diff
@@ -166,7 +166,7 @@ def read_subscription(subscription_id)
             "#{redis_key(SUBSCRIPTION_PREFIX)}#{subscription_id}",
             :query_string, :variables, :context, :operation_name
           ).tap do |subscription|
-            next if subscription.values.all?(&:nil?) # Redis returns hash with all nils for missing key
+            return if subscription.values.all?(&:nil?) # Redis returns hash with all nils for missing key
```

Comment:
Looks like tests fail due to this `return` (and RuboCop is not happy).

Let's refactor to use `.then { |subscription| next if ..; ...; subscription }`

Reply:
I've pushed a new commit using then instead of tap